.github/workflows/entropy-check.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: entropy-check
+
+on:
+  pull_request_target:
+    branches:
+      - main
+  workflow_dispatch:
+
+
+jobs:
+  entropy-check:
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install python env
+        uses: ./.github/actions/install-python-env
```

Comment:
Consider adding an explicit parameter for the `fetch-depth` to help indicate how this job requires a certain fetch depth. Doing this may help developers recall what's happening with the data used by the almanack below.